### ISSUE TITLE
Incompatible with the latest discord.py

### ISSUE THREAD

**Issue by TomSparkLabs** [fritzr/discord-twitter-bot#5]
This bot is incompatible with the latest discord.py, and as such, is unusable for the majority of people who have other bots running alongside it.

**Comment by fritzr** [comment]
Yes, I expected this to happen soon - we wrote this bot right as they were
developing discord v0.10 and had announced the API would not be reverse
compatible. If the new release had been available we would've designed for
it. I'll update to v0.10 eventually™. Thanks for letting me know the new
interface is out.
